stable/sonarqube/templates/test-config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sonarqube.fullname" . }}-tests
```

Comment:
Add standard labels.

Reply:
Do you have an example of what I am missing here?

Reply:
```yaml
labels:
  app: {{ template "sonarqube.name" . }}
  chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
  release: {{ .Release.Name }}
  heritage: {{ .Release.Service }}
```

Reply:
Added